RULES_EXPAND

```diff
@@ -1,8 +1,8 @@
 #
 # RULES_EXPAND
 # Creates build rules needed to expand templated IOC config files
-#
-EXPAND       = /reg/g/pcds/controls/macro/expand
+RULES_EXPAND := $(lastword $(MAKEFILE_LIST))
+EXPAND       := $(dir $(RULES_EXPAND))/expand
```

Comment:
Looks like a more dynamic version that doesn't tie us to `/reg/g` but instead infers it from the `MAKEFILE_LIST` variable

Reply:
Yep.